spec/support/vcr_cassettes/lighthouse/veteran_verification/status/200_show_response.yml

```diff
@@ -0,0 +1,58 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://sandbox-api.va.gov/services/veteran_verification/v2/status/1012667145V762142
```

Comment:
Please safeguard the uri. ` <URL>/services/veteran_verification/v2/status/1012667145V762142`